code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -35,21 +35,11 @@
 	//Temperature effect
 	var/minbodytemp = 250
 	var/maxbodytemp = 350
-	var/heat_damage_per_tick = 3	//amount of damage applied if animal's body temperature is higher than maxbodytemp

```

Comment:
I don't think people actually edit those two vars on a live server very often, plus they can still modify minbodytemp and maxbodytemp to change a mob's reaction to different temperatures.